lib/ansible/modules/web_infrastructure/sentry_project.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Mikhail Naletov <okgolove@markeloff.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: sentry_project
+short_description: Manage Sentry project
+description:
+    - Manage the state of a project in Sentry.
+version_added: "2.8"
+
+options:
+  api_token:
+    description:
+      - API token generated in Sentry. The token sufficient access to manipulate projects.
+    required: true
```

Comment:
You should always provide the `type` here. (After `description`, before `required` or `default`.)
```suggestion
    type: str
    required: true
```